bin/rbenv-each

```diff
@@ -13,21 +13,25 @@ set -e
 [ -n "$RBENV_DEBUG" ] && set -x
 
 # Provide rbenv completions
-if [ "$1" = "--complete" ]; then
-  echo -v
```

Comment:
eh, maybe. I'm not usually a fan of including short-flags in tab completion. since they're only 2 chars and the first char is common with all flags, the second char must always be typed. so tab completion doesn't actually help, and in fact harms the tab completion for long flags